addon/components/ember-popper-base.js

```diff
@@ -28,34 +19,38 @@ export default Component.extend({
 
   /**
    * Modifiers that will be merged into the Popper instance's options hash.
-   * https://popper.js.org/popper-documentation.html#Popper.DEFAULTS
+   * https://popper.js.org/docs/v2/modifiers
    * @argument
    * @type(optional('object'))
    */
-  modifiers: null,
+  /* eslint-disable ember/avoid-leaking-state-in-ember-objects */
+  modifiers: [],
 
   /**
-   * onCreate callback merged (if present) into the Popper instance's options hash.
-   * https://popper.js.org/popper-documentation.html#Popper.Defaults.onCreate
+   * onFirstUpdate callback merged (if present) into the Popper instance's options hash.
+   * https://popper.js.org/docs/v2/lifecycle/#hook-into-the-lifecycle
    * @argument
    * @type(optional(Function))
    */
-  onCreate: null,
+  onFirstUpdate: null,
 
   /**
-   * onUpdate callback merged (if present) into the Popper instance's options hash.
-   * https://popper.js.org/popper-documentation.html#Popper.Defaults.onUpdate
-   * @argument
-   * @type(optional(Function))
+   * Placement of the popper. One of ['auto', 'top', 'right', 'bottom', 'left'].
+   * Each placement can have a variation from this list: ['-start', '-end']
+   * https://popper.js.org/docs/v2/constructors/#placement
+   * @argument({ defaultIfUndefined: true })
+   * @type('string')
    */
-  onUpdate: null,
```

Comment:
`onUpdate` callback is gone https://popper.js.org/docs/v2/migration-guide/#9-update-callbacks